### PR TITLE
Update path in ALCC example conf files

### PR DESCRIPTION
**Pull request by xylar**
The name of the E3SM project has changed.

### CODE REVIEW

**Comment by stephenprice** [comment]
@xylar -- I've got a job in the queue now based on an edited version of these scripts. If that works, I'll review / approve this.

File: configs/alcf/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta [hunk 1]
```diff
@@ -39,13 +39,13 @@ ncclimoParallelMode = bck
...
 # when running "download_analysis_data.py" to get the public observations,
 # mapping files and region files.
-baseDirectory = /lus/theta-fs0/projects/ClimateEnergy_2/diagnostics
+baseDirectory = /lus/theta-fs0/projects/ClimateEnergy_3/diagnostics
 
 [input]
 ## options related to reading in the results to be analyzed
 
 # directory containing model results
-baseDirectory = /lus/theta-fs0/projects/OceanClimate_2/mpeterse/20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta/run
```
Comment (stephenprice) [inline configs/alcf/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta]: To make it even more confusing, ClimateEnergy switched from "_2" to "_3" but OceanClimate switched from nothing to "_2". So these should go back to "OceanClimate_2" in each example script.

Comment (xylar) [reply configs/alcf/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta]: Sorry about that.  Now fixed.

**Comment by milenaveneziani** [comment]
I had to do something like this to run the analysis on theta just today, and everything works fine.

**Comment by xylar** [comment]
@milenaveneziani, did you run on `theta` or `cooley` login nodes? I'm able to run there but we got busted for doing that in the past so I recommend not doing so.  I can't get the code to run on `cooley` compute nodes because of an HDF5 error I don't understand.

**Comment by milenaveneziani** [comment]
I am running on cooley compute nodes, with no problems. Here is the job I am using:
`~milena/MPAS-git-repositories/MPAS-Analysis/job_script.cooley.bash`

**Comment by milenaveneziani** [comment]
That script uses the conda package. I believe I was having the same HDF5 error when running with `mpas_analysis...`

**Comment by xylar** [comment]
Thanks @milenaveneziani.  So that suggests that the error was introduced by something changed in MPAS-Analysis since v1.1.  That's odd, since I don't see why any changes we've made would affect the places in the code where the errors are coming from but it's still a very helpful clue.

**Comment by milenaveneziani** [comment]
we could probably merge this, right?

**Comment by xylar** [comment]
@stephenprice is still working on this.  He's going to update the config files.

**Comment by stephenprice** [comment]
@xylar -- I added the updated (working and tested) analysis configs for the 30to10wLI and the 18to6. I also add the edited job scripts, one for each (due to the diff. settings needed to get them to complete successfully). The 30to10wLI analysis is [here](http://portal.nersc.gov/project/m2833/mpas_analysis_output/20190301.GMPAS-DIB-IAF-ISMF.T62_oRRS30to10v3wLI.theta/ocean/index.html). The 18to6 analysis is [here](http://portal.nersc.gov/project/m2833/mpas_analysis_output/oRRS18to6v3.theta.TEST/).

**Comment by xylar** [comment]
Thanks @stephenprice!

**Pull request merged**